### ISSUE TITLE
editor: "Open" project

### ISSUE THREAD

**Issue by gmarinov** [vizorvr/vizor#1777]
could do with a clarification of what this does and how it looks

**Comment by mattatgit** [comment]
New & Open button behaviours are in the proto we put up a while back for the Welcome dialogue.
https://app.atomic.io/d/ctKlujdOKjeG

**Comment by esaruoho** [comment]
@mattatgit  
<img width="447" alt="error_404" src="https://user-images.githubusercontent.com/4966687/27899632-b1d67ed4-6233-11e7-8785-016f81e474e0.png">

**Comment by mattatgit** [comment]
@esaruoho 
Yeah, that's been archived & i don't know which one exactly it refers to. We can't keep this stuff alive forever unfortunately.
